website/static/js/pages/login-page.js

```diff
@@ -5,7 +5,6 @@
 var $ = require('jquery');
 
 var passwordForms = require('js/passwordForms');
-var InstitutionSignIn = require('js/institutionSignIn');
```

Comment:
Is what this viewmodel was doing now handled by CAS? If so, consider removing dead code in `institutionSignIn.js`